azure-iot-device/azure/iot/device/common/chainable_exception.py

```diff
@@ -15,7 +15,5 @@ def __init__(self, message=None, cause=None):
         self.__cause__ = cause
         super(ChainableException, self).__init__(message)
 
-    def __repr__(self):
-        return "{} caused by {}".format(
-            super(ChainableException, self).__repr__(), self.__cause__.__repr__()
-        )
+    def __str__(self):
+        return "{} caused by {}".format(super(ChainableException, self).__str__(), self.__cause__)
```

Comment:
good change.

Reply:
+1

---
In reply to: [337131174](https://github.com/Azure/azure-iot-sdk-python/pull/365#discussion_r337131174) [](ancestors = 337131174)